json_parsers/json_parser2.py

```diff
@@ -0,0 +1,36 @@
+import json
+
+path="D:\\python\\input.json"
```

Comment:
```suggestion
Please read the input.json file from the path  automation-frameworks/json_files/  instead of hardcoding 
path="D:\\python\\input.json" 
```